puppet/modules/quickstack/manifests/cinder_volume.pp

```diff
@@ -89,9 +103,9 @@
         Class['::cinder::volume']
       }
 
-      if ($::selinux != "false") {
+      if ($::selinux != false) {
```

Comment:
I am not sure this is a safe change.  It appears [1] that as of puppet 4, facts with return as data types other than string, but I do not believe our release is expect to run on that new a version, which could cause this to not return as expected.

[1] https://docs.puppetlabs.com/puppet/latest/reference/lang_facts_and_builtin_vars.html#handling-boolean-facts-in-older-puppet-versions